### PR TITLE
correct formatting issue with highstate output module

### PR DESCRIPTION
**Pull request by leifliddy**
### What does this PR do?
Corrects a formatting issue when ```cmd.run``` is run from an orchestration and returns multiple lines of output. This PR ensures every line of output under ```Changes``` is indented properly.

### What issues does this PR fix or reference?
#56554

### Previous Behavior

given the orchestration state

```
command_1:
  salt.function:
    - name: cmd.run
    - tgt: 'server'
    - arg:
      - ls /tmp
```
the files listed under ```Changes``` would not be indented properly
```
[root@server ~]# salt-run state.orch orch.test --out highstate
server.example.com_master:
----------
          ID: command_1
    Function: salt.function
        Name: cmd.run
      Result: True
     Comment: Function ran successfully. Function cmd.run ran on server.
     Started: 19:02:44.195687
    Duration: 413.993 ms
     Changes:   
              server:
                  file1
              file2
              file3
```

### New Behavior
```
[root@server ~]# salt-run state.orch orch.test --out highstate
server.example.com_master:
----------
          ID: command_1
    Function: salt.function
        Name: cmd.run
      Result: True
     Comment: Function ran successfully. Function cmd.run ran on server.
     Started: 19:03:19.668603
    Duration: 418.08 ms
     Changes:   
              server:
                  file1
                  file2
                  file3
```

### CODE REVIEW

File: salt/output/highstate.py [hunk 1]
```diff
@@ -186,13 +186,18 @@ def _format_host(host, data, indent_level=1):
...
+    if data is None or data is True or data is False:
+        nchanges = 1
+        hstrs.append(('{0}    {1}{2[ENDC]}'.format(hcolor, data, colors)))
+        hcolor = colors["CYAN"]  # Print the minion name in cyan
+    elif isinstance(data, int) or isinstance(data, six.string_types):
         # Data in this format is from saltmod.function,
         # so it is always a 'change'
         nchanges = 1
-        hstrs.append(("{0}    {1}{2[ENDC]}".format(hcolor, data, colors)))
+        for data in data.splitlines():
```
Comment (DmitryKuzmenko) [inline salt/output/highstate.py:197]: The `int` type has no attribute `splitlines`.

Comment (DmitryKuzmenko) [reply salt/output/highstate.py:197]: Possibly you've meant to handle `int` in the first block handling `None` and `bool`?

**Review by DmitryKuzmenko — CHANGES_REQUESTED** [review]

**Comment by DmitryKuzmenko** [comment]
@leifliddy could you please write a unit test for it?

File: salt/output/highstate.py (hunk 1 above)
Comment (leifliddy) [reply salt/output/highstate.py:197]: That was oversight on my part.
Odd, I never realized that True and False are numerically equivalent to  1 and 0 in python
So, the first check just needs to be
```if isinstance(data, int) or data is None:```

In the future, if someone wanted to apply different colors to different data types, then it might make sense to do a ```if data is True or data is False```  check. 
But, there's no need to perform that check in the current module (it's a bit wordy anyway) 

Ok, I've just pushed the new commit, this one should pass no problem. 

> could you please write a unit test for it?
I'm not sure how to do that, I'll research that a bit later.

Comment (leifliddy) [reply salt/output/highstate.py:197]: I'm going to close this PR and submit a new one, the current PR is too messy given the relative simplicity of the change requested.

**Comment by leifliddy** [comment]
closing PR, will open a new one with less commits.

File: salt/output/highstate.py (hunk 1 above)
Comment (DmitryKuzmenko) [reply salt/output/highstate.py:197]: It's okay. Thank you for your work!

**Comment by DmitryKuzmenko** [comment]
@leifliddy thank you for your work. Let me know if you need any help on this.

File: salt/output/highstate.py (hunk 1 above)
Comment (leifliddy) [reply salt/output/highstate.py:197]: the new PR# is 56598
The ```highstate.py``` file changed since my last pull, which is the reason for the 2nd commit.

**Comment by leifliddy** [comment]
the new PR# is 56598

**Comment by DmitryKuzmenko** [comment]
Adding a link to associate them to each other #56598
